Add `osu!taiko Mapping World Cup 2026` news post and wiki article (#14853)
ppy/osu-wiki: wiki/Contests/TMWC/2026/en.md

---
tags:
  - o!tmwc
  - tmwc
---

# osu!taiko Mapping World Cup 2026

![contest banner](img/banner.jpg)

The **osu!taiko Mapping World Cup 2026** (***TMWC 2026***) is an upcoming country-based, multi-stage osu!taiko mapping contest that aims to crown which nation is the best at creating osu!taiko beatmaps.

## Schedule

| Event | Timestamp (UTC) |
| --: | :-- |
| Round 1 mapping phase | July 11 until August 1 |
| Round 1 judging phase | August 1 until August 22 |
| Round 2 mapping phase | August 22 until September 20 |
| Round 2 judging phase | September 20 until October 10 |
| Round 3 mapping phase | October 10 until November 8 |

> | Round 3 judging phase | November 8 until November 28 |

Reviewer: ```suggestion
| Round 1 mapping phase | July 11 – August 1 |
| Round 1 judging phase | August 1 – August 22 |
| Round 2 mapping phase | August 22 – September 20 |
| Round 2 judging phase | September 20 – October 10 |
| Round 3 mapping phase | October 10 – November 8 |
| Round 3 judging phase | November 8 – November 28 |
```

| Final results | November 28 |

Please note this schedule is subject to change.

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 55% of total cash prize[^cash-prize], unique profile badge, [3 contest points](/wiki/Contests/Contest_points) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of total cash prize[^cash-prize], [2 contest points](/wiki/Contests/Contest_points) |

> | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 15% of total cash prize[^cash-prize], [1 contest point](/wiki/Contests/Contest_points) |

Reviewer: this is unneeded if you're explaining it below the table anyway

```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 55% of total cash prize, unique profile badge, [3 contest points](/wiki/Contests/Contest_points) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of total cash prize, [2 contest points](/wiki/Contests/Contest_points) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 15% of total cash prize, [1 contest point](/wiki/Contests/Contest_points) |
```

> The amount of contest points awarded will be determined by the number of mappers participating in the contest, as detailed in the [contest points key](/wiki/Contests/Contest_points#points-key). The cash prize will be crowd-funded and those who wish to support the contest can do so [here](https://ko-fi.com/ikin5050).

Reviewer: ```suggestion
The amount of contest points awarded will be determined by the number of mappers participating in the contest, as detailed in the [contest points key](/wiki/Contests/Contest_points#points-key). The cash prize will be crowd-funded and those who wish to support the contest can do so via [ikin5050's Ko-fi page](https://ko-fi.com/ikin5050).
```

## Organisation

The osu!taiko Mapping World Cup 2025 is run by various community members.

| Position | Member(s) |
| :-- | :-- |
| Host | ::{ flag=NL }:: [ikin5050](https://osu.ppy.sh/users/4007649), ::{ flag=AT }:: [Yasuho](https://osu.ppy.sh/users/8458835),  ::{ flag=PT }:: [Warp](https://osu.ppy.sh/community/users/18649724) |
| Designers | ::{ flag=MY }:: [Jerry](https://osu.ppy.sh/users/605973) |
| Judges | ::{ flag=HK }:: [Cynplytholowazy](https://osu.ppy.sh/users/3901754), ::{ flag=DE }:: [OnosakiHito](https://osu.ppy.sh/users/290128), ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669), ::{ flag=GB }:: [Skidooskei](https://osu.ppy.sh/users/10079029), ::{ flag=VN }:: [davidminh0111](https://osu.ppy.sh/users/9623142),  ::{ flag=DE }:: [Undead Alice](https://osu.ppy.sh/users/17415683), ::{ flag=FR }:: [Heaxys](https://osu.ppy.sh/users/5671417), ::{ flag=PH }:: [Kulalaying](https://osu.ppy.sh/users/1259391) |

## Links

- Announcement news post
- [Forum post](https://osu.ppy.sh/community/forums/topics/2215635?n=1)
- [Contest listing](https://osu.ppy.sh/community/contests/287)
- [Discord server](https://discord.com/invite/5ewzEEcUCB)

## Ruleset

### Format

> - Round 1 will act as a seeding round. Round 2 will feature a knockout round according to the seeding from Round 1. The top teams from each group will progress to Round 3 (the final). Every team will get to participate in Rounds 1&2. Teams will be expected to produce 1 map per round.

Reviewer: ```suggestion
- Round 1 will act as a seeding round. Round 2 will feature a knockout round according to the seeding from Round 1. The top teams from each group will progress to Round 3 (the final). Every team will get to participate in Rounds 1 and 2. Teams will be expected to produce 1 map per round.
```

### Rules

- **The [osu! community rules](/wiki/Rules) are in place at all times during this contest.**
- **The [osu!taiko ranking criteria](/wiki/Ranking_criteria/osu!taiko) and the [general ranking criteria](/wiki/Ranking_criteria) are in effect for this contest.** No submission may be intentionally unrankable. Mistakes can happen, but submissions that cannot be ranked without major changes will be disqualified.
- **A team must consist of 3–4 players from the same country.** For countries that do not have enough members to participate, teams will be formed by the organisation team based on geographical location. If insufficient players from a country sign up, the hosts will suggest ways to combine with registrants from another country based on geographical proximity. You are allowed to sign up as a premade team but the hosts reserve the right to ask for someone to join your premade team to ensure every signup can participate.
- **At least one (1) team member must join the contest's [Discord server](https://discord.com/invite/5ewzEEcUCB)**

> - **At least two (2) mappers must collaborate per song.** This is a team-based competition. Collaboration is necessary to determine the strengths of each participating country. We want to avoid situations amongst teams where some members get ‘carried’. Collaboration does not include anything other than mapping (notes/scroll speed changes/volume changes), providing feedback does not count.

Reviewer: ```suggestion
- **At least two (2) mappers must collaborate per song.** This is a team-based competition. Collaboration is necessary to determine the strengths of each participating country. We want to avoid situations amongst teams where some members get "carried". Collaboration does not include anything other than mapping (notes/scroll speed changes/volume changes), providing feedback does not count.
```

- **We expect everyone to conduct themselves with a sporting attitude and with integrity. Respect toward fellow competitors, judges and organisers is expected at all times. If you share your creations outside of your team channel before judging is complete, or attempt any form of leaking confidential information, you will be disqualified.**

### Judging criteria

- **Flow/Playability (20):** Points will be assigned based on a clear correlation between hand movements and the song, as well as how the map plays. Difficulty curve and distribution in relation to the song progression will be assessed too.
- **Structure and representation(20):** Points will be assigned based on how well the song is represented through your mapping choices, consistency, and how well the collaborative mapping aspect was handled.
- **Judge Impression (20):** Points will be assigned based on the judge's opinion of your submission.
- **Creativity (20):** Points will be assigned based on how the submission uses creative and fitting elements to make itself stand out.

Scores will be reported as standardised scores.

## Notes

> [^cash-prize]: The cash prize pool is generously crowdfunded by the osu!taiko community.

Reviewer: ```suggestion
```
